app/views/samples/_form.haml

```diff
@@ -65,22 +65,6 @@
           = f.number_field :volume, min: 0, step: :any, :class => "input-small", readonly: !@can_update
           .span.unit (μl)
 
-      = f.form_field :reference_gene do
-        - @sample_form.blinded_attribute(:reference_gene) do
-          = autocomplete_field(f, "sample[reference_gene]", :reference_gene, @navigation_context.institution, class: "input-x-large")
-
-      = f.form_field :target_organism_taxonomy_id do
-        - @sample_form.blinded_attribute(:target_organism_taxonomy_id) do
-          = autocomplete_field(f, "sample[target_organism_taxonomy_id]", :target_organism_taxonomy_id, @navigation_context.institution, class: "input-x-large")
-
-      = f.form_field :pango_lineage do
-        - @sample_form.blinded_attribute(:pango_lineage) do
-          = autocomplete_field(f, "sample[pango_lineage]", :pango_lineage, @navigation_context.institution, class: "input-x-large")
-
-      = f.form_field :who_label do
-        - @sample_form.blinded_attribute(:who_label) do
-          = autocomplete_field(f, "sample[who_label]", :who_label, @navigation_context.institution, class: "input-x-large")
-
```

Comment:
**question**: institutions that received samples won't be able to view these fields anymore (they can't access the batches), except from the NIH tables. Is it expected?

Reply:
These fields for samples never arrived to production, so there will be no impact in removing this.